docs/dependency-specification.md

```diff
@@ -145,6 +145,20 @@ flask = { git = "https://github.com/pallets/flask.git", rev = "38eb5d3b" }
 numpy = { git = "https://github.com/numpy/numpy.git", tag = "v0.13.2" }
 ```
 
+In cases where the package you want to install is located in a subdirectory of the VCS repo, you can use the subdirectory option:
+
+```toml
+[tool.poetry.dependencies]
+# Install a package named `subdir_package` from a folder called `subdir` within the repo
+subdir_package = { git = "https://github.com/mypackage_with_subdirs.git", subdirectory = "subdir" }
+```
+
+When using `poetry add` to achive the same options, you can use the same URL formatting that is also supported by [pip](https://pip.pypa.io/en/stable/topics/vcs-support/#url-fragments).
```

Comment:
```suggestion
with the corresponding `add` call:
```
to be consistent with https://github.com/python-poetry/poetry/blob/a3aafa840de950f81e99553e52190a54c94d6bce/docs/dependency-specification.md?plain=1#L213 and https://github.com/python-poetry/poetry/blob/a3aafa840de950f81e99553e52190a54c94d6bce/docs/dependency-specification.md?plain=1#L249